src/client/utils/image.js

```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line import/prefer-default-export
+export const buildImageUrl = ({ id, pictureId, quality = 'medium' }) =>
+  id && pictureId
+    ? `https://graasp.eu/pictures/${id}/${quality}_${pictureId}`
```

Comment:
You might want to factor out the base_url too, in order to make it more flexible.

Reply:
I wanted to factor it out using some env variables but it's not possible in the front end for now. But I can factor it out with some constants for now.